stubs/whichcraft/whichcraft.pyi

```diff
@@ -0,0 +1,5 @@
+__author__: str
+__email__: str
+__version__: str
+
+def which(cmd: str, mode: int = ..., path: str | None = ...) -> str | None: ...
```

Comment:
We now prefer to include default values where possible, but, again, our script to autogenerate stubs hasn't yet been updated to reflect our recent change in policy there :)

```suggestion
def which(cmd: str, mode: int = 1, path: str | None = None) -> str | None: ...
```